.eslintrc

```diff
@@ -1,14 +1,14 @@
 {
-    "plugins": ["@typescript-eslint"], // Add
+    "plugins": ["@typescript-eslint"],
     "extends": [
-      // "next/babel",
       "next",
       "next/core-web-vitals",
       "plugin:@typescript-eslint/recommended",
       "prettier"
     ],
     "rules": {
-      "@typescript-eslint/no-unused-vars": "error",
```

Comment:
Why did you remove that check?

Reply:
when working on nextauth, there was one callback function for redirection that had parameters that were not used, and it kept showing me errors, that's y I removed it, coz i wasn't using them anywhere, I have removed the code it was for testing